frontend/src/metabase/search/containers/SearchApp.jsx

```diff
@@ -81,7 +81,7 @@ function SearchApp({ location }) {
       <Search.ListLoader query={query} wrapped>
         {({ list, metadata }) => (
           <SearchBody direction="column" justify="center">
-            <SearchControls>
+            <SearchControls pb="lg">
```

Comment:
I must have missed previous PRs when this got changed, but now when I add/change/remove filters in the search results page, the whole page disappears to show the loading state. I'd expect that only results disappear, and filter inputs are still shown when results are being reloaded.

Reply:
yep there's another conversation going on about this one too